src/jit/gtlist.h

```diff
@@ -97,6 +97,10 @@ GTNODE(SIMD_CHK         , GenTreeBoundsChk   ,0,GTK_SPECIAL|GTK_NOVALUE)// Compa
                                                                         // does the compare, so that it can be more easily optimized.  But that involves generating qmarks at import time...
 #endif // FEATURE_SIMD
 
+#ifdef FEATURE_HW_INTRINSICS
+GTNODE(HWIntrinsic_CHK  , GenTreeBoundsChk   ,0,GTK_SPECIAL|GTK_NOVALUE)
```

Comment:
All the node names are all-caps. This shouldn't depart from that convention. I would suggest HW_INTRINSIC_CHK

Reply:
@CarolEidt, I think this also deviated with `GenTreeHWIntrinsic`. Should a bug be logged to fix that as well?

Reply:
Ah yes, I didn't even realize that. GT_HWIntrinsic should probably be GT_HW_INTRINSIC. Thanks